apps/neuvector/crds/kustomization.yaml

```diff
@@ -1,8 +1,4 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - https://raw.githubusercontent.com/neuvector/manifests/main/kubernetes/5.4.0/crd-k8s-1.19.yaml
```

Comment:
- not sure if manifest URLs can be removed. 
- also version declared in crds-neuvetor.yaml does not match `5.4.0` appeared in the manifest urls.

I recall traefik2 is also installed via CRD and it clearly says `update the crd version in traefik-crds when updating this`
https://github.com/hmcts/cnp-flux-config/blob/e6e8681a1b6cd7a63e5bb209fdc47cffaca6ba91/apps/admin/traefik2/traefik2.yaml#L12
the version `35.2.0` matches the one appears in the manifest urls 
https://github.com/hmcts/cnp-flux-config/blob/e6e8681a1b6cd7a63e5bb209fdc47cffaca6ba91/apps/admin/traefik-crds/kustomization.yaml#L4

Reply:
Regarding the manifests, they are now not needed because we are no longer applying CRDs manually using raw manifests, we are using the official Neuvector chart repo which means that Helm is now responsible for installing these CRD's.